data/mil2525d/utilities/military-features-utilities/template-gdb-toolbox/scripts/CreateTemplateGDB/UpdateDomains.py

```diff
@@ -0,0 +1,59 @@
+# ----------------------------------------------------------------------------------
```

Comment:
I don't think I noticed this before - why is there an extra **CreateTemplateGDB** subfolder under `scripts` - full path = `template-gdb-toolbox\scripts\` **CreateTemplateGDB** - seems like these should be moved up a level